src/main/java/yoda/YodaUI.java

```diff
@@ -0,0 +1,158 @@
+package yoda;
+import yoda.task.Task;
+import yoda.task.TaskList;
+import yoda.storage.Storage;
+
+import java.io.IOException;
+
+
+public class YodaUI {
+    private final TaskList taskList;
+    private final String chatbotName;
+    private boolean isChatting;
+    private final Storage storage;
```

Comment:
```suggestion
    private final TaskList TASKLIST;
    private final String CHATBOT_NAME;
    private boolean isChatting;
    private final Storage STORAGE;
```

constants should be uppercase with underscore to separate words.